apps/condo/migrations/20221013121613-0177_assigneescope_assigneescopehistoryrecord_and_more.js

```diff
@@ -0,0 +1,518 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+--
+SET statement_timeout = '1500s';
+
+--
+-- [CUSTOM]
+--
+CREATE EXTENSION if not exists pgcrypto;
```

Comment:
we already have it here: `20210707112100-00027-insert_default_organization_employee_roles.js`
Do we really need it again?

Reply:
removed